src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,189 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    private static final int CAR_ID_INDEX = 1;
+    private static final int MANUFACTURER_NAME_INDEX = 6;
+    private static final int DRIVER_ID_INDEX = 11;
+    private static final int DRIVER_NAME_INDEX = 12;
+    private static final int DRIVER_LICENCE_NUMBER_INDEX = 13;
+
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (model, manufacturer_id)"
+                + " VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement =
+                        connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, car.getModel());
+            statement.setLong(2, car.getManufacturer().getId());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create car: " + car, e);
+        }
+        try {
+            insertDrivers(car);
+            return car;
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT * FROM cars c"
+                + " JOIN manufacturers m ON c.manufacturer_id = m.id"
+                + " JOIN car_drivers cd ON c.id = cd.car_id"
+                + " JOIN drivers d ON d.id = cd.driver_id"

```

Comment:
Instead of using the same join in all queries, I'd recommend you put the logic of getting drivers to the additional method and reuse it. also it's improve your method of parsing from ResultSet, cause better to get a data by colomn name instead of colomn id